<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuizCsharp</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-10</y>
      <width>551</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="startLevel3Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>128</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startLevel2Button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>129</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startLevel1Button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>129</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitLevel1Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitLevel2Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>264</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitLevel3Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень 1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
